plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevControllerExtensions.kt

```diff
@@ -88,6 +92,14 @@ suspend fun FeatureDevController.onCodeGeneration(session: Session, message: Str
 
         messenger.sendCodeResult(tabId = tabId, uploadId = uploadId, filePaths = filePaths, deletedFiles = deletedFiles, references = references)
 
+        if (remainingIterations != null && totalIterations != null) {
+            messenger.sendAnswer(
+                tabId = tabId,
+                messageType = FeatureDevMessageType.Answer,
+                message = "You have $remainingIterations out of $totalIterations code iterations remaining."
```

Comment:
Can we add the message in the strings .properties file? It allows variables there?

Reply:
Updated the message with .properties.